src/ert/gui/ertwidgets/suggestor_message.py

```diff
@@ -0,0 +1,52 @@
+import textwrap
+
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import QHBoxLayout, QLabel, QWidget
+
+
+class SuggestorMessage(QWidget):
+    def __init__(self, lbl_text, color, text):
+        QWidget.__init__(self)
+        common_style = """
+        border-style: outset;
+        border-width: 2px;
+        border-radius: 10px;
+        border-color: beige;
```

Comment:
I just wanted to comment on this. The outset isn't really visible when you blend white and beige on the border.
It's no biggie, but maybe not the outset effect you want. I've only tested on windows 11, and rhel 11, but ubuntu seem to show similar results also.
If you replace the `beige` with `darkgrey` you see what i mean.

Reply:
`darkgrey` makes it pop :sparkling_heart: